keyboards/handwired/g_split_board/readme.md

```diff
@@ -0,0 +1 @@
+A split ergonomic keyboard with enough key's for national layout
```

Comment:
This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).